package.json

```diff
@@ -102,7 +102,7 @@
     "source-sans-pro": "^2.0.10",
     "spin.js": "^2.3.2",
     "string_score": "^0.1.22",
-    "swagger-client": "2.1.15",
+    "swagger-client": "2.1.13",
```

Comment:
Rebase error :) Fixed.
Needs more testing on swagger client: I have one differing behavior on badges (thanks to the tests)